ci/no-superuser/create-no-superuser-sqlserver.sh

```diff
@@ -21,7 +21,12 @@ do
 
     echo "INFO: Retry count: ${COUNT}"
 
-    docker exec -t ${SQL_SERVER_CONTAINER_NAME} /opt/mssql-tools/bin/sqlcmd -S localhost -U sa -P ${SQL_SERVER_PASSWORD} -d master -Q "SELECT 1"
+    if [[ "$1" = "sqlserver22" ]]; then
```

Comment:
For readability, how about extracting the common part into a function like this?

```bash
function sqlcmd () {
    if [[ "$1" = "sqlserver22" ]]; then
        docker exec -t ${SQL_SERVER_CONTAINER_NAME} /opt/mssql-tools18/bin/sqlcmd -S localhost -U $2 -P $3 -d $4 -C -Q $5
    else
        docker exec -t ${SQL_SERVER_CONTAINER_NAME} /opt/mssql-tools/bin/sqlcmd -S localhost -U $2 -P $3 -d $4 -Q $5
    fi
}
```

Reply:
Another option might be defining a variable for the path to `sqlcmd` above based on the container name?

```shell
if [[ "$SQL_SERVER_CONTAINER_NAME" = "sqlserver22" ]]; then
  SQLCMD=/opt/mssql-tools18/bin/sqlcmd
else
  SQLCMD=/opt/mssql-tools/bin/sqlcmd
fi

...

docker exec -t ${SQL_SERVER_CONTAINER_NAME} ${SQLCMD} -S localhost
```

Reply:
Ah, I see. Looks good!
I will update the script! Thank you for your suggestion!

Reply:
@brfrn169 @komamitsu 
I fixed it in 218608a6d17495fae60091b7c9699cf7fc90efc4 and 51428be622c36f99545c17831fadab540bc0b523 .
(Sorry, I overlooked @komamitsu 's suggestion. So, I updated it twice.😇 But finally, I adopted `using variables` to switch two versions of `sqlcmd`.)